### ISSUE TITLE
Very confused by the docs

### ISSUE THREAD

**Issue by on3iro** [FormidableLabs/spectacle#886]
### Question

Hey guys, I just wanted to get started with spectacle, but the docs are confusing me quite a bit.
These are the issues i am having:

1. "Getting Started with Development" - this paragraph tells me, that all my source files have to be inside the `src/` dir. So far so good. But this also reads a bit like there should already be such a directory after installing spectacle with `yarn/npm`. I assume spectacle just wants me to create `src` myself?
2. The next part tells me to start a development server with ominous npm-scripts which are not yet part of my package.json. Instead I should look inside `examples/js` - I realize these are part of the spectacle repository. But how am I supposed to access these just after running `yarn add spectacle`? Ofcourse I could look inside my node_modules, but is this what you want users to actually do?
3. what is `spectacle-boilerplate`? You mention it multiple times inside the docs, but without a link where it could be installed from. However you use it directly after a cli prompt `$ spectacle-boilerplate` which suggests, that it should already be installed with something else. Do I need to install spectacle-cli to get spectacle-boilerplate?
5. In general it would be nice to have some kind of quick start guide. Aka "follow these few steps to get a running setup".
4. Is there a way to use spectacle with multiple markdown files? I can't seem to find anything inside the docs

Thanks in advance :)

**Comment by kale-stew** [comment]
Hi Theo! Thanks for opening this issue. We spent a lot of time rewriting these docs and issues like these help us better understanding where we might have missed our mark. 🙂 

1. It seems like our use of the phrase "Getting Started" in two places is confusing. The "Getting Started with Development" paragraph you initially refer to here is about _developing_ against the Spectacle src. I agree, this blurb should probably be moved into a Contributing section or somewhere else, where it's more obvious that this is about the dev process and not deck creation. If you navigate to "Getting Started" (second from the bottom in the lefthand navigation), you will find the tutorial we wrote to introduce newcomers to the library.

2. By "the next part" are you referring to the "Writing Your Presentation" sub section? We _did_ link to the `spectacle-cli` repository and documentation directly in this section, but I understand if the link decoration wasn't obvious at first read. We intended to make it more clear that use of `spectacle-cli` was just another option for developing with Spectacle, but it sounds like we missed that mark with our phrasing as well. To address the second half of your question here, it sounds like we should once again better indicate that "Getting Started: A Tutorial" will be your better bet for clear, step-by-step instructions, and not the "Basic Concepts" subsections.

3. I don't believe we have a process documented anywhere for using multiple markdown or jsx files, but I do know we have a couple of open issues surrounding imported slides (#871, #870) so we will need to resolve the existing bugs before we formalize a process around imported slides. I can spend some time on this today.

Hopefully I've addressed your concerns here, but please don't hesitate to ask more clarifying questions. Additionally, I will try to reorg our docs to make them more intuitive to navigate. I'm sure you're not the only one with these questions. 👍

**Comment by on3iro** [comment]
@kale-stew Hi Kylie - thanks a lot for the quick response.

Yeah, I think re-organizing the docs a little bit would make them definitely more intuitive to read. I suspected that the `basic concepts` section was more like a contribution section, but wasn't sure about it. I will definitely have a look at the issues you mentioned. I am currently transferring a huge amount of slides from a keynote into a spectacle presentation and putting everything into a single file makes organization really difficult.

A few more smalle questions:

1. Is there a way to apply a background color to multiple slides (I currently do use the basic deck which was created by running `spectacle-boilerplate` and modify it to my needs. However I could not seem to find a way to overwrite the default background color.)
2. Is it possible to apply styles to markdown slides?

Thanks again :)

**Comment by on3iro** [comment]
@kale-stew one more question ;) - is there a way to define the default indentation level of `<CodePane>`s? Currently it looks like it indents a single space inside the presentation for each 2 spaces inside my actual code block, which is pretty annoying.

**Actual Code**:
![Screenshot 2020-04-27 at 08 55 15](https://user-images.githubusercontent.com/8681413/80342686-dcc18a80-8864-11ea-8ae7-9143e80df58f.png)

**Presentation**:
<img width="329" alt="Screenshot 2020-04-27 at 08 55 22" src="https://user-images.githubusercontent.com/8681413/80342697-e2b76b80-8864-11ea-94b9-dcbfde3b5259.png">

**Comment by slorber** [comment]
Did you find a way to customize the background color easily using theme or template?

I don't find any solution

https://github.com/FormidableLabs/spectacle/issues/938

**Comment by on3iro** [comment]
@slorber unfortunately I didn't (if I remember correctly - it's been a while since I last used spectacle)

**Comment by Zenahr** [comment]
I agree on how the docs are confusing to read. Thanks for opening this issue. I might look into contributing to the docs as well and maybe we could get it more streamlined for first-time users by seperating the docs into dev-docs and user-docs.
